File: 250624/아래로 사각형 채우기/filling-rectangle-with-downflow.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int n;
    cin >> n;
    int cnt = 1;
    int arr[10][10];
    for (int i=0; i<n; i++) {
        for (int j=0; j<n; j++) {
            arr[j][i] = cnt;
            cnt+=1;
        }
    }

    for (int i=0; i<n; i++) {
        for (int j=0; j<n; j++) {
            cout << arr[i][j] << " ";
        }
        cout << "\n";
    }
    return 0;
}

File: 250627/아래로 사각형 채우기/filling-rectangle-with-downflow.py
n = int(input())
cnt = 1

arr = [[0 for _ in range(n)] for _ in range(n)]
for i in range(n):
    for j in range(n):
        arr[j][i] = cnt
        cnt +=1

for i in arr:
    for j in i:
        print(j,end=" ")
    print("")